.vscode/launch.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
Is this file need to be included in the git?

Reply:
Sorry, my hands slipped...